### PR TITLE
Add vitamin a information to the artifact

### PR DESCRIPTION
**Pull request by KjellSwedin**

### CODE REVIEW

File: src/vivarium_conic_lsff/data/builder.py [hunk 1]
```diff
@@ -210,3 +210,22 @@ def load_and_write_lbwsg_data(artifact: Artifact, location: str):
...
+
+
+def load_and_write_vitamin_a_deficiency_data(artifact: Artifact, location: str):
+    keys = [
+        project_globals.VITAMIN_A_DEFICIENCY_CATEGORIES,
+        project_globals.VITAMIN_A_DEFICIENCY_RESTRICTIONS,
+        project_globals.VITAMIN_A_DEFICIENCY_EXPOSURE,
+        project_globals.VITAMIN_A_DEFICIENCY_RELATIVE_RISK,
+        project_globals.VITAMIN_A_DEFICIENCY_PAF,
+        project_globals.VITAMIN_A_SEQ_DW_MODERATE_VI,
```
Comment (collijk) [inline src/vivarium_conic_lsff/data/builder.py]: So our model is going to just be susceptible or with condition.  We only need the cause level disability weight, which you should be able to retrieve directly with get_measure

Comment (collijk) [reply src/vivarium_conic_lsff/data/builder.py]: also, there's no mortality in this model, yeah?
Maybe write a comment here to that effect.

**Review by collijk — CHANGES_REQUESTED** [review]
Use cause level disability

File: src/vivarium_conic_lsff/data/builder.py (hunk 1 above)
Comment (KjellSwedin) [reply src/vivarium_conic_lsff/data/builder.py]: Done and done.

**Review by collijk — APPROVED** [review]

**Pull request merged**